pymc3/backends/base.py

```diff
@@ -541,8 +541,7 @@ def point(self, idx, chain=None):
             If a chain is not given, the highest chain number is used.
         """
         if chain is None:
-            chain = self.chains[-1]
```

Comment:
This looks like it will break a lot of things internally... 

Surprisingly, it sounds like it was a deliberate design choice to iterate over the last chain in a MultiTrace. Therefore, we cannot just make it illegal without providing an alternative interface / making changes in the methods that depend on this functionality.

Just for fun I'll let the run tests to see how quickly it fails.